cmd/node-cache/main.go

```diff
@@ -176,28 +185,35 @@ func (c *cacheApp) parseAndValidateFlags() error {
 func (c *cacheApp) runChecks() {
 	for _, rule := range c.iptablesRules {
 		exists, err := c.iptables.EnsureRule(utiliptables.Prepend, rule.table, rule.chain, rule.args...)
-		if !exists {
-			if err != nil {
-				cache.teardownNetworking()
-				clog.Fatalf("Failed to add back non-existent rule %v", rule)
-			}
+		switch {
+		case exists:
+			// debug messages can be printed by including "debug" plugin in coreFile.
+			clog.Debugf("IP table rule (table %q, chain %q) already exists", rule.table, rule.chain)
+			continue
+		case err == nil:
 			clog.Infof("Added back nonexistent rule - %v", rule)
```

Comment:
Added nodelocaldns rule to table %q, chain %q

current log message is hard to understand unless you already know what is going on

Reply:
Printing the rule which have the table/chain info as well. Modified the log message wording as suggested.